### ISSUE TITLE
False positive "Duplicate spaces" for match

### ISSUE THREAD

**Issue by akalongman** [slevomat/coding-standard#1340]
For match expressions like:
```php
$mail = match ($number) {
    '123456789' => '123456789@mail.ge',
    '12345'         => '12345@mail.ge',
    '1234'           => '1234@mail.ge',
    default         => $mail,
};
```
shows error: "Duplicate spaces at position xx. (SlevomatCodingStandard.Whitespaces.DuplicateSpaces.DuplicateSpaces)"

My config looks like:
```
<rule ref="SlevomatCodingStandard.Whitespaces.DuplicateSpaces">
    <properties>
        <property name="ignoreSpacesBeforeAssignment" value="true"/>
        <property name="ignoreSpacesInParameters" value="true"/>
        <property name="ignoreSpacesInAnnotation" value="true"/>
        <property name="ignoreSpacesInComment" value="true"/>
    </properties>
</rule>
```

**Comment by kukulich** [comment]
I don’t see any error.

**Comment by kukulich** [comment]
Do you think it should be resolved as “assigment” ?

**Comment by akalongman** [comment]
Yes, like multiline arrays, or should be added new parameter "IgnoreSpacesInMatch"
